### PR TITLE
add Markdown version of INSTALL (v2)

### PR DESCRIPTION
**Pull request by larsnaesbye**
A resubmit of the previous pull request after I failed to squash properly.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+* Check https://celestia.space/download.html.
+* Check https://bintray.com/celestia if it contains packages for your system.
+
+Stable version installation on Windows and OSX:
+* Check https://celestia.space/download.html.
+
+Development snapshots installation on Unix-like systems:
+### On Debian 10 and derived systems:
+
+  `curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key`
```
Comment (375gnu) [inline INSTALL.md]: single backtick is for inline code, for blocs of code use \`\`\`

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+As said prevously Celestia provides several user interfaces, accordingly with
+interfaces it's built with it has different executable files installed to
+${CMAKE_INSTALL_PREFIX}/bin (e.g. with default CMAKE_INSTALL_PREFIX on
+Unix-like systems they are installed into `/usr/local/bin`).
+
+Here's the table which provides executable file names accordingly to interface:
+
+ Interface | Executable name
+-  | :-
```
Comment (375gnu) [inline INSTALL.md]: still garbage rendered

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+* Check https://celestia.space/download.html.
+
+Development snapshots installation on Unix-like systems:
+### On Debian 10 and derived systems:
+
+```
+curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key
+sudo apt-key add -echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/ ./" 
+sudo tee /etc/apt/sources.list.d/celestia-obs.list
+sudo apt update && sudo apt install celestia
```
Comment (375gnu) [inline INSTALL.md]: this block is brocken, it should be:
```
  curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key | sudo apt-key add -
  echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/ ./" | sudo tee /etc/apt/sources.list.d/celestia-obs.list
  sudo apt update && sudo apt install celestia
```

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+```
+
+### On Ubuntu 18.04 and derived systems:
+
+```
+curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Ubuntu_18.04/Release.key
+sudo apt-key add -echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Ubuntu_18.04/ ./"
+sudo tee /etc/apt/sources.list.d/celestia-obs.list
+sudo apt update && sudo apt install celestia
```
Comment (375gnu) [inline INSTALL.md]: As above

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+
+  Try experimental portable AppImage (see https://github.com/CelestiaProject/Celestia/issues/333):
+```
+wget https://github.com/munix9/Celestia/releases/download/continuous/celestia-qt-continuous-x86_64.AppImage
+chmod 755 celestia-qt-continuous-x86_64.AppImage
+```
+
+  Optionally create a portable, version-independent `$HOME` directory in the same folder as the AppImage file:
+```
+mkdir celestia-qt.home`
```
Comment (375gnu) [inline INSTALL.md]: extra backtick at the end of line

File: INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,360 @@
...
+   This option effect is overriden by NATIVE_OSX_APP.  
+ \*\* Ignored on Windows systems.  
+ \*\*\* Ignored on Unix-like systems.  
+ \*\*\*\* This option support is not finished yet.  
+
+Parameters of type "bool" accept ON or OFF value. Parameters of type "path"
+accept any directory.
+
+On Windows systems two additonal options are supported:
+- `CMAKE_GENERATOR_PLATFORM`  |- can be set to x64 on 64-bit Windows to build
```
Comment (375gnu) [inline INSTALL.md]: plz remove ` |`

**Comment by larsnaesbye** [comment]
Should be fixed, and squashed.

File: INSTALL.md (hunk 3 above)
Comment (375gnu) [reply INSTALL.md]: you still have 2 lines written as one: `sudo apt-key add -echo ...`. `echo` should on the next line

File: INSTALL.md (hunk 4 above)
Comment (375gnu) [reply INSTALL.md]: still not fixed

File: INSTALL.md [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+```
+
+### On openSUSE Tumbleweed:
+
+```
+sudo zypper addrepo https://download.opensuse.org/repositories/home:munix9:unstable/openSUSE_Tumbleweed/home:munix9:unstable.repo
+sudo zypper refresh
+sudo zypper install celestiagit rebase -i HEAD~2
+replace the 2nd pick with f
+git push -f
```
Comment (375gnu) [inline INSTALL.md]: please remove:
git rebase -i HEAD~2
replace the 2nd pick with f
git push -f

File: INSTALL.md [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+
+Clone the source and go to the source directory.
+
+Configure and build:
+
+```
+mkdir build
+cd build
+cmake -DENABLE_SPICE=ON ..
+make -jN`
```
Comment (375gnu) [inline INSTALL.md]: extra backtick after jN

**Comment by eyvallah** [comment]
I think we can delete the current INSTALL file, also make README.md link to INSTALL.md rather than INSTALL

**Comment by larsnaesbye** [comment]
Would you like that as part of this pull request as well?

**Comment by eyvallah** [comment]
@larsnaesbye I think so

**Pull request merged**